LeadKit/LeadKit/Extensions/Alamofire/AlamofireManager+Extensions.swift

```diff
@@ -52,6 +52,19 @@ public extension Reactive where Base: Alamofire.SessionManager {
             .flatMap { $0.validate().rx.apiResponse(mappingQueue: mappingQueue) }
     }
 
+    /// Method which executes request and serializes response into array of target objects
+    ///
+    /// - Parameter requestParameters: api parameters to pass Alamofire
+    /// - Parameter mappingQueue: The dispatch queue to use for mapping
+    /// - Returns: Observable with HTTP URL Response and array of target objects
+    func responseModel<T: ImmutableMappable>(requestParameters: ApiRequestParameters,
+                       mappingQueue: DispatchQueue = DispatchQueue.global())
+        -> Observable<(HTTPURLResponse, [T])> {
+
+        return apiRequest(requestParameters: requestParameters)
+            .flatMap { $0.rx.apiResponse(mappingQueue: mappingQueue) }
```

Comment:
I did it in the same way as regular mapping (to single T). What is the difference?